config/runtime/tests/ltp.jinja2

```diff
@@ -0,0 +1,15 @@
+
+- test:
+    name: {{ node.name }}
+    timeout:
+      minutes: {{ job_timeout }}
+    definitions:
+       - repository: https://github.com/kernelci/test-definitions
+         from: git
+         revision: kernelci.org
+         path: automated/linux/ltp/ltp.yaml
+         name: {{ node.name }}
+         parameters:
+            TST_CMDFILES: "{{ tst_cmdfiles }}"
```

Comment:
```suggestion
            TST_CMDFILES: "{{ tst_cmdfiles|default('') }}"
```

Without this we're forced to add the `tst_cmdfiles`, while it can be optional IIUC